types/simulation/types.go

```diff
@@ -158,7 +157,7 @@ type AppParams map[string]json.RawMessage
 // object. If it exists, it'll be decoded and returned. Otherwise, the provided
 // ParamSimulator is used to generate a random value or default value (eg: in the
 // case of operation weights where Rand is not used).
-func (sp AppParams) GetOrGenerate(_ codec.JSONCodec, key string, ptr interface{}, r *rand.Rand, ps ParamSimulator) {
+func (sp AppParams) GetOrGenerate(key string, ptr interface{}, r *rand.Rand, ps ParamSimulator) {
```

Comment:
This seems unnecessary. I mean this parameter isn't needed, but we should avoid API breakage when it's non-essential

Reply:
i do agree, but i think this release will have lots of breaking things its fine we add something small like this